data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/damage_type/damage_type_attacker_defender_filtering.cfg

```diff
@@ -0,0 +1,80 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: apply_to=both with [filter_attacker][filter_weapon]type= and [filter_defender][filter_weapon]type=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# change type of attack of pierce and resistance to -50%.
+# Give bob's weapon specials [damage_type]applied to attacker and active if attacker and defender attack are type=pierce.
+# Have Alice attack with his weapon.
+##
+# Expected end state:
+# Deterministic end state, without crashing, but BROKE STRICT.
+# Damage_type is active
+# Bob takes 15 damage.
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "damage_type_applied_with_attacker_and_defender_filtering" (
```

Comment:
Change the test name to "damage_type_apply_to_attacker_filter_attacker_defender".

Reply:
done